disco_aws_automation/disco_alarm.py

```diff
@@ -10,6 +10,7 @@
 
 # Max batch size for alarm deletion http://goo.gl/vMQOrX
 DELETE_BATCH_SIZE = 100
+ENVIRONMENT_DELETE_SKIP_NAMESPACES = ['AWS/ES']
```

Comment:
We dont delete ES cluster at moment because we didnt have backup of ES. Pretty soon we will have and we might want to delete the ES cluster along with VPC spin down. @amplifylitco/astronauts : What do you guys think about spinning down ES cluster along with the VPC. There can be clusters that might be specific to a service which wont serve any purpose when the service is down.